lib/mathlib.cpp

```diff
@@ -268,6 +268,9 @@ MathLib::value MathLib::value::shiftLeft(const MathLib::value &v) const
     if (!isInt() || !v.isInt())
         throw InternalError(nullptr, "Shift operand is not integer");
     MathLib::value ret(*this);
+    if (v.intValue > 63) {
```

Comment:
Tests were performing UB.
For example test/testvalueflow.cpp:464 was still running bit shift, which is UB.

Reply:
Would throwing an exception make more sense?

Reply:
hmm.. if exception is thrown then there is a danger we won't check the code at all. returning some arbitrary value seems ok.

Reply:
Fixed as you said.